common/src/lib.rs

```diff
@@ -44,3 +47,28 @@ pub fn generate_kafka_client_config(
     }
     client_config
 }
+
+pub fn create_default_consumer(
+    broker_address: &String,
+    username: &Option<String>,
+    password: &Option<String>,
+    consumer_group: &String,
+    topics_to_subscribe: Option<&[&str]>,
```

Comment:
Does `topics_to_subscribe: Option<&[&str]>` need to be `Option`? What happens when `consumer.subscribe` is given an empty list? I don't think that case will ever arise anyway as a consumer should always subscribe to at least one topic.

Reply:
That makes sense, I'm not sure why I made it `Option` to begin with.